metrics/src/main/scala/com/readytalk/metrics/WorkbenchStatsD.scala

```diff
@@ -0,0 +1,24 @@
+package com.readytalk.metrics
+
+import com.typesafe.scalalogging.LazyLogging
+
+/**
+  * Created by rtitle on 8/21/17.
+  */
+class WorkbenchStatsD(host: String, port: Int) extends StatsD(host, port) with LazyLogging {
+  // Filter out detailed timer metrics to prevent hitting Hosted Graphite quotas.
+  // Keep: mean, p95, stddev
+  val MetricSuffixesToFilter = Set("max", "min", "p50", "p75", "p98", "p99", "p999", "samples", "m1_rate", "m5_rate", "m15_rate", "mean_rate")
```

Comment:
Unfortunately these are all hard-coded into metrics-statsd, so this seems like the best/only way to filter it.

https://github.com/ReadyTalk/metrics-statsd/blob/master/metrics3-statsd/src/main/java/com/readytalk/metrics/StatsDReporter.java#L204-L219